README.rst

```diff
@@ -35,3 +35,18 @@ Depending on your editor, the configuration (found in a file called mypy-ls.cfg
 	"live_mode": True,
 	"strict": False
     }
+
+``dmypy`` (default is False) executes via `dmypy run` rather than `mypy`.
+
+This uses the `dmypy` daemon and may dramatically improve the responsiveness of the `pyls` server.
```

Comment:
[python-lsp-server](https://github.com/python-lsp/python-lsp-server) uses `pylsp`, not `pyls`.

Reply:
```suggestion
This uses the `dmypy` daemon and may dramatically improve the responsiveness of the `pylsp` server.
```